### PR TITLE
bug: `wrangler pages functions` does not log a beta warning

### PR DESCRIPTION
**Pull request by threepointone**
In https://github.com/cloudflare/wrangler2/pull/661, we wrote a test, but didn't actually test if the error gets thrown. Being explicit about the thrown error shows that the  error doesn't get thrown, and the warning doesn't get logged.

--- 

@JacobMGEvans Feel free to add commits to this branch and make the test pass.

### CODE REVIEW

**Review by petebacondarwin — COMMENTED** [review]
I'm not sure an error should be thrown, right?
We should just be checking what the `stderr` contains?

**Comment by threepointone** [comment]
Sure, yeah. It still doesn't log the message from what I see.

File: packages/wrangler/src/__tests__/pages.test.ts [hunk 1]
```diff
@@ -51,32 +51,38 @@ describe("subcommand implicit help ran on imcomplete command execution", () => {
...
         Must specify a directory of static assets to serve or a command to run.]
       `);
-    }
   });
 
   it("should display for pages:functions", async () => {
+    let err: Error | undefined;
     try {
-      execaSync("npx", ["wrangler", "pages", "functions"], {
+      execaSync("npx", ["wrangler", "pages", "functions", "build"], {
```
Comment (threepointone) [inline packages/wrangler/src/__tests__/pages.test.ts:73]: to clarify, we should be able to get the message for just `wrangler pages functions`, unless that's not actually possible?

Comment (JacobMGEvans) [reply packages/wrangler/src/__tests__/pages.test.ts:73]: It's not possible, it's a subcommand with no output.

**Comment by threepointone** [comment]
Thanks!

**Pull request merged**